.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -237,14 +248,15 @@ jobs:
           fi
 
       - run: nextstrain build . ${{ inputs.build-args }}
+        working-directory: ${{ inputs.build-dir }}
 
       - if: always()
         uses: actions/upload-artifact@v3
         with:
           name: ${{ inputs.artifact-name }}-${{ matrix.runtime }}
           path: |
-            auspice/
-            results/
-            benchmarks/
-            logs/
-            .snakemake/log/
+            ${{ inputs.build-dir }}/auspice/
+            ${{ inputs.build-dir }}/results/
+            ${{ inputs.build-dir }}/benchmarks/
+            ${{ inputs.build-dir }}/logs/
+            ${{ inputs.build-dir }}/.snakemake/log/
```

Comment:
There's back-and-forth here among the 3 commits - I think they can be squashed into a single commit.

Reply:
For sure